### PR TITLE
Add guard utility runner

### PR DESCRIPTION
**Pull request by alexwllms**
Includes, guard-reload, rspec runner, rails start runner and guard-bundler.

Running 'guard' or `bundle exec guard` will start rails, live-reload and run rspec on file change. Pressing `enter` while guard is running will "manually" run the rspec tests.

@benlovell @lshepstone @jongilbraith enjoy.

### CODE REVIEW

**Comment by benlovell** [comment]
I bloody hate guard.

**Comment by alexwllms** [comment]
Other than `guard-rspec` what are the downsides to having it as part of the codebase?

File: Guardfile [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# Note: The cmd option is now required due to the increasing number of ways
+#       rspec may be run, below are examples of the most common uses.
+#  * bundler: 'bundle exec rspec'
+#  * bundler binstubs: 'bin/rspec'
+#  * spring: 'bin/rspec' (This will use spring if running and you have
+#                          installed the spring binstubs per the docs)
+#  * zeus: 'zeus rspec' (requires the server to be started separately)
+#  * 'just' rspec: 'rspec'
+
+guard :rspec, cmd: "bundle exec rspec" do
```
Comment (benlovell) [inline Guardfile]: Quotes.

File: Guardfile [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  require "guard/rspec/dsl"
```
Comment (benlovell) [inline Guardfile]: Quotes.

Give this file a quick once over for quoting please.

**Comment by benlovell** [comment]
@alexwllms Nothing really. If *you're* going to use it I've no problem with this being merged.

**Comment by jongilbraith** [comment]
Following up on what @benlovell, :+1: I guess. :smile:

**Comment by alexwllms** [comment]
I'm taking that as two thumbs. What a bunch of divas. ROFL.

**Pull request merged**